### DISCUSSION TITLE
Building model using own dataset

### DISCUSSION THREAD

**Discussion by SrinivasKalagotla** [pyannote/pyannote-audio#1203] [Q&A]
Hi, I have a 5min long audio file of a person, I want to use this audio to create a model and then use this model for speaker diarization. I want to know how can I do it.

I am using Google's UIS-RNN speaker diarization algorithm in my present work in which I extract train-sequences from the audio file (d-vectors) and train the model with these sequences.

I want to know If I can do the same procedure with pyannote as well. Please help me

**Comment by hbredin** [comment]
The [documentation](https://github.com/pyannote/pyannote-audio#documentation) has a dedicated section on how to adapt a pipeline to your own data.
